cmd/operator-sdk/add/api.go

```diff
@@ -125,25 +127,32 @@ func apiRun(cmd *cobra.Command, args []string) error {
 		&scaffold.Register{Resource: r},
 		&scaffold.Doc{Resource: r},
 		&scaffold.CR{Resource: r},
-		&scaffold.CRD{Resource: r, IsOperatorGo: projutil.IsOperatorGo()},
 	)
 	if err != nil {
 		return fmt.Errorf("api scaffold failed: (%v)", err)
 	}
 
+	gcfg := genutil.Config{InputDir: scaffold.ApisDir, OutputDir: scaffold.CRDsDir}
+	crd := gencrd.NewCRDGo(gcfg)
+	if err := crd.Generate(); err != nil {
+		return errors.Wrapf(err, "error generating CRDs from APIs in %s", scaffold.ApisDir)
+	}
+
+	log.Info("Generated CustomResourceDefinition manifests.")
+
```

Comment:
Do we need this code since we're running it in `k8sgenutil.OpenAPIGen()`?

Reply:
We do not, especially with #2276.